pcl_ros/package.xml

```diff
@@ -24,35 +23,23 @@
   <url type="bugtracker">https://github.com/ros-perception/perception_pcl/issues</url>
   <url type="repository">https://github.com/ros-perception/perception_pcl</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
-  <build_depend>cmake_modules</build_depend>
-  <build_depend>rosconsole</build_depend>
-  <build_depend>roslib</build_depend>
-
-  <depend>dynamic_reconfigure</depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
   <depend>eigen</depend>
-  <depend>message_filters</depend>
-  <depend>nodelet</depend>
-  <depend>nodelet_topic_tools</depend>
   <depend>libpcl-all-dev</depend>
   <depend>pcl_conversions</depend>
-  <depend>pcl_msgs</depend>
-  <depend>pluginlib</depend>
-  <depend>rosbag</depend>
-  <depend>roscpp</depend>
+  <depend>rclcpp</depend>
   <depend>sensor_msgs</depend>
-  <depend>std_msgs</depend>
-  <depend>tf</depend>
-  <depend>tf2_eigen</depend>
+  <depend>geometry_msgs</depend>
+  <depend>tf2</depend>
+  <depend>tf2_geometry_msgs</depend>
+  <depend>tf2_ros</depend>
 
-  <test_depend>rostest</test_depend>
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
+  <test_depend>ament_cmake_gtest</test_depend>
 
   <export>
-    <nodelet plugin="${prefix}/plugins/nodelet/libpcl_ros_features.xml"/>
```

Comment:
Might be good to comment these out so they can be added back later easier

Reply:
Agreed, will do. I originally decided to take the 'start clean, add stuff when needed' instead of keeping old ROS1 cruft around, but since I've added all the ROS1 targets back to the CMakeLists, I should also keep this around but commented.

Reply:
done